malcolm/modules/builtin/controllers/managercontroller.py

```diff
@@ -125,12 +125,19 @@ def _run_git_cmd(self, *args):
                     ("git",) + args, cwd=self.config_dir)
             except subprocess.CalledProcessError as e:
                 self.log.warning("Git command failed: %s\n%s", e, e.output)
+                return None
             else:
                 self.log.debug("Git command completed: %s", output)
+                return output
 
     def do_init(self):
         super(ManagerController, self).do_init()
         # Try and make it a git repo, don't care if it fails
+        name = self._run_git_cmd("config", "--get", "user.name")
+        if name == "\n":
+            self._run_git_cmd("config", "user.name", "Malcolm")
```

Comment:
Can we make it take the username and machine?

```suggestion
import getpass
import socket
            self._run_git_cmd("config", "user.email", "%s@%s" % (getpass.getuser(), socket.gethostname())
```